Makefile

```diff
@@ -11,6 +11,7 @@ clean: coverageclean typecoverageclean
 
 typecheck:
 	mypy --cobertura-xml-report typecover --html-report typecover .
+	mypy tests/integration/records
```

Comment:
I wanted to use types to document complex parts of the integration tests like this, and of course validate those annotations, but not have it count against the metrics.